### DISCUSSION TITLE
0.0.18: llava-v1.5 support?

### DISCUSSION THREAD

**Discussion by Katehuuh** [turboderp/exllamav2#446] [Q&A] [answered]
> [0.0.18](https://github.com/turboderp/exllamav2/releases/tag/v0.0.18) Changelog: `- Very preliminary multimodal support`

So exl2 does support [llava-v1.5](https://huggingface.co/models?search=llava%20exl)?

**Comment by turboderp** [comment, marked as answer]
Yes, in a limited sense. You still need the vision tower from Transformers to produce the image feature embeddings, but now there's an interface for mixing the embeddings into the prompt. See #399 for more.

It will hopefully become a full set of features at some point, maybe even dropping the Transformers dependency. Just different priorities right now.
